programs/range/src/main.rs

```diff
@@ -108,7 +109,8 @@ fn main() {
                 println!("cycle-tracker-end: oracle-load");
 
                 println!("cycle-tracker-report-start: oracle-verify");
-                oracle.verify().expect("key value verification failed");
+                // TODO: EigenDA verification
+                // oracle.verify().expect("key value verification failed");
```

Comment:
Correct, you should only be commenting out the blob related logic here: https://github.com/mantle-xyz/op-succinct/blob/fbc15cfb054363f1c90fcf470a2e3c6313ed98a9/utils/client/src/oracle/mod.rs#L130-L200, as you're now posting to EigenDA. 

When you implement verification, it will go in this function.

Reply:
makes sense. Restored oracle verification logic and commented out the GlobalGeneric-related code, pending future implementation.
https://github.com/mantle-xyz/op-succinct/blob/mantle-kona-zkvm-test/utils/client/src/oracle/mod.rs#L123